extensions/wikia/ExactTargetUpdates/maintenance/ExactTargetUpdateUserEditsPerWikiMaintenance.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Maintenance script for updating number of edits made by user on a wiki
+ * in ExactTarget database
+ *
+ * @package MediaWiki
+ * @addtopackage maintenance
+ *
+ * @author Kamil Koterba <kamil@wikia-inc.com>
+ *
+ */
+
+require_once( __DIR__ . "/../../../../maintenance/commandLine.inc" );
+
+class ExactTargetUpdateUserEditsPerWikiMaintenance {
+
+	private $aBotsList = NULL;
+
+	/**
+	 * Maintenance script entry point.
+	 * Gathering user edits data from last day and adding update task to job queue.
+	 * Script skips old TempUser relicts and bot accounts.
+	 */
+	public function execute() {
+		global $wgDWStatsDB;
+		// Get DB
+		$oStatsDBr = wfGetDB( DB_SLAVE, array(), $wgDWStatsDB );
+		$sStartDate = $this->getLastDayDate();
+		$oUsersListResult = $this->getUsersEditedRecently( $oStatsDBr, $sStartDate );
+		$aUsersEditsData = $this->getUserEdits( $oStatsDBr, $sStartDate, $oUsersListResult );
+		$this->addEditsUpdateTask( $aUsersEditsData );
+	}
+
+	private function getUsersEditedRecently( DatabaseBase $oStatsDBr, $sStartDate ) {
+		$timeCondition = $this->prepareTimeCondition( $sStartDate );
+		// Get list of users that made edits in last period
+		$oUsersListResult = $oStatsDBr->query("SELECT distinct user_id from rollup_wiki_user_events where {$timeCondition} and user_id != 0;");
```

Comment:
What's the reasoning for writing these queries raw rather than using the standard `Database::select` or FluentSQL? It should at the very least pass a `__METHOD__` so we can track the query when debugging.

This line is also too long per our coding conventions.